distribution/scripts/jmeter/performance-test-scenarios.sh

```diff
@@ -66,6 +69,7 @@ declare -A test_scenario2=(
 declare -A test_scenario3=(
     [name]="CBRTransportHeaderProxy"
     [display_name]="CBRTransportHeaderProxy"
+    [description]="Reading transport header and route to backend"
```

Comment:
Routing the message based on a HTTP header in the message

Reply:
Changed